_libs/nav/head.js

```diff
@@ -137,29 +224,55 @@ const navItems = [
           "Iterative Models",
           "Hyperparameter Optimization",
         ],
-      },
-      {
-        name: "AMES",
+        ilos: [
+          "Understand the different types and methods introduced by MLJ",
+          "Refresh your skills on building simple models",
+          "Learn how to prepare example real-life data by loading, coercing, partitioning and unpacking data",
+          "Learn how to build pipelines in MLJ",
+          "Learn about how to manually and automatically evaluate models in MLJ",
+          "Understand how to perform feature selection in MLJ",
+          "Learn how to wrap models in iterative strategies in MLJ",
+          "Learn how to tune hyperparameters in MLJ",
+          "Learn how to save and perform final evaluations on your models in MLJ"
+        ]
```

Comment:
something about ROC curves and confusion matrices. and if you could get "stratified cross-validation" in there, that would be great.

Reply:
Also, I'd  move the  first `ilos` to last last, as it's really only mentioned as a summary for reference and not a key attraction.

I know these are learning objectives, but the *key* point about this tutorial is the audience: it's for practicing data scientists wanting to quickly transition to Julia. So if you could get that in somehow.

Reply:
Done.